android/src/main/java/com/dooboolab/RNIap/RNIapModule.java

```diff
@@ -522,8 +522,8 @@ public void onPurchasesUpdated(BillingResult billingResult, @Nullable List<Purch
         item.putBoolean("isAcknowledgedAndroid", purchase.isAcknowledged());
         item.putInt("purchaseStateAndroid", purchase.getPurchaseState());
 
+        promiseItem = item.copy();
```

Comment:
the `copy` method is only added in RN 0..61. Should probably bump up the major version and update doc to reflect that.

Reply:
@mars-lan Hey! Thank you for the detail. Um... I should have to think about it since I feel kind of reluctant to separate the major version. @rewieer How do you think?

Reply:
The copy method basically calls merge, so we can create a `public WriteableMap copy(WritableMap original)`.

Reply:
@rewieer Sounds great! Would you like to give a `PR` for your idea?

Reply:
@hyochan sorry for the long delay, I was about to do it but it seems someone did it before me